query/Makefile

```diff
@@ -19,11 +19,12 @@ HAIL_REVISION := $(shell git rev-parse HEAD)
 ifeq ($(NAMESPACE),default)
 JAR_LOCATION := gs://hail-query/jars/$(HAIL_TEST_GCS_TOKEN)/$(HAIL_REVISION).jar
 else
-JAR_LOCATION := gs://hail-test-dmk9z/$(HAIL_TEST_GCS_TOKEN)/jars/$(HAIL_REVISION).jar
+JAR_LOCATION := gs://hail-test-dmk9z/$(NAMESPACE)/jars/$(HAIL_REVISION).jar
```

Comment:
If you look at config.mk, I just shell out to get this field from k8s (don't love it but it's the best I could come up with at the time). Could do the same thing here. I basically asserted that anyone using these makefiles is a dev and is authenticated with k8s